herddb-utils/src/main/java/herddb/utils/ExtendedDataInputStream.java

```diff
@@ -29,9 +29,14 @@
  * Extended version of DataInputStream
  *
  * @author enrico.olivelli
+ * @author diego.salvi
+ *
+ * @see ExtendedDataOutputStream
  */
 public class ExtendedDataInputStream extends DataInputStream {
 
+    private static final boolean USE_FOLDED_VAR_INT = Boolean.getBoolean("herddb.vint.read.folded");
```

Comment:
can we use SystemProperties.getBooleanSystemProperty and set the default to "true" ?